### ISSUE TITLE
docs: add `mdx` ending to `tailwind.config.js`'s `content` array file paths to watch

### ISSUE THREAD

**Issue by becknik** [nativewind/nativewind#1038]
**Describe the bug**

I'm not sure if this is really is necessary, but I ran into an issue today which nearly drove me crazy because of the `mdx` files not being processed by tailwind. I'm not sure why this is even necessary, because my project doesn't contain any `.mdx` files. But when running on node-js 22.5.1 with this dependencies:

```
├── expo-router@3.5.23
├── expo@51.0.32
├── nativewind@4.1.6
├── tailwindcss@3.4.10
```

I'm getting the following errors on some reloads:

```
(node:73054) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
```

And this in combination with a further bug lead into some hours of bug-hunting...

Just reporting this, so that some other people running into this thing. Decide yourself, if this is something to adjust, or if its some weird stuff only happening on my machine...

**Comment by danstepanov** [comment]
@becknik thanks for flagging, can you repro using create expo stack? https://github.com/nativewind/nativewind/blob/main/contributing.md#opening-an-issue

**Comment by becknik** [comment]
I just decided to ignore this document, since my "bug" is rather a question if it makes sense to add the `mdx` to the doc or not.
I have not enough knowledge in expo, fast reload or any other technology to reason if so.

However, during the previous bug-fighting, I have tried to use your Stackblitz template to file an issue to figure out what's wrong with my project setup since I followed the instructions. I couldn't get to the point where I could run the `expo start -c` and see, if the fast reload on save works in there.
